src/vs/workbench/contrib/testing/browser/testingExplorerView.ts

```diff
@@ -136,7 +136,7 @@ export class TestingExplorerView extends ViewPane {
 	}
 
 	public override shouldShowWelcome() {
-		return this.viewModel?.welcomeExperience === WelcomeExperience.ForWorkspace ?? true;
+		return this.viewModel ? this.viewModel.welcomeExperience === WelcomeExperience.ForWorkspace : true;
```

Comment:
@connor4312 Please check that this is the correct fix. The `?? true` was unreachable before

Reply:
It is reachable. The viewModel is marked as not null !` and is assigned in `renderBody` (because practically it's defined in every other usage in the view) but `shouldShowWelcome` can be called before view bodies are rendered.

Reply:
The precedence of `??` means the expression is actually evaluated as:

```ts
(this.viewModel?.welcomeExperience === WelcomeExperience.ForWorkspace) ?? true;
```

Since the `===`  always returns boolean, the `?? true` has no effect